src/extension.ts

```diff
@@ -894,8 +894,8 @@ export function findKindNameOrPrompt(resourceKinds: kuberesources.ResourceKind[]
     }
 }
 
-function promptKindName(resourceKinds: kuberesources.ResourceKind[], descriptionVerb, opts, handler) {
-    vscode.window.showInputBox({ prompt: "What resource do you want to " + descriptionVerb + "?", placeHolder: 'Empty string to be prompted' }).then((resource) => {
+function promptKindName(resourceKinds: kuberesources.ResourceKind[], descriptionVerb, opts, handler, placeHolder: string = 'Empty string to be prompted') {
```

Comment:
I think this placement is problematic.  The `handler` is likely to be an inline code block representing the core of the calling function; the placeHolder could therefore end up separated by 10-20 lines from the call to `promptKindName`, so it is very hard for the reader to connect this random string back to the role it plays.  An alternative might be to add it to the `opts` record?

Reply:
not sure I got you here will need more clarity on this fo fix neatly :)